libs/constrained-generators/src/Constrained/Spec/Map.hs

```diff
@@ -210,10 +239,10 @@ instance
                     )
         explain1 ("  n = " ++ show n) $ go n kvs mempty
   genFromTypeSpec (MapSpec mHint mustKeys mustVals size (simplifySpec -> kvs) foldSpec) = do
-    mustMap <- explain1 "Make the mustMap" $ forM (Set.toList mustKeys) $ \k -> do
-      let vSpec = constrained $ \v -> satisfies (pair_ (lit k) v) kvs
+    !mustMap <- explain1 "Make the mustMap" $ forM (Set.toList mustKeys) $ \k -> do
```

Comment:
What's the motivation for all these bangs?

Reply:
I think those are left over from some debugging use trace. I removed them